examples/systemcalls/systemcalls.c

```diff
@@ -17,7 +17,11 @@ bool do_system(const char *cmd)
  *   or false() if it returned a failure
 */
 
-    return true;
+	int ret = system(cmd);
+	if(ret == -1)
+		return false;
+	else
+    	return true;
```

Comment:
@MuktaDarekar consider what would happen if the command you issued returned a negative error return value other than -1.  Would you meet the requirements in the function code block?